README.md

```diff
@@ -8,6 +8,26 @@ The Seguro Platform application.
 yarn
 ```
 
+Notice:
+
+There are two dependencies (`"@conet-project/seguro-gateway": "0.15.2"`,
+`"@conet-project/seguro-worker-lib": "^0.10.1"`) are not published in npm repository, so when you install those two
+dependencies, you should follow the steps below:
+
+1. clone the them in your local disk by using `git clone` command, and those two packages code url
+   are: `https://github.com/CoNET-project/seguro-worker-lib.git`
+   and `https://github.com/CoNET-project/seguro-gateway.git`
+
```

Comment:
2. install both projects with `npm install`

Reply:
Thanks for your review, I will do it later